src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,138 @@
+package duke.task;
+
+import duke.exception.DukeException;
+
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * TaskList contains the list that is storing the current tasks.
+ * Mainly deal with the addition, deletion and manipulation of tasks in the list.
+ */
+public class TaskList {
+    private final List<Task> list;
```

Comment:
Perhaps a variable name that reflects the things contained in the list, e.g. tasks?